assets/src/components/developers/RunKit.ts

```diff
@@ -88,18 +88,22 @@ async function handler({event, payload}) {
 
 async function handleMessageCreated(message) {
   const {body, customer_id, conversation_id} = message;
-  const isCustomerMessage = !!customer_id;
 
-  if (isCustomerMessage && body.toLowerCase().startsWith('test')) {
+    messageBody = body.toLowerCase()
+    let responseMessage = 'This is the default response'
+    
+    if(messageBody.includes('test')) {
+      responseMessage = "test successful!"
+    } else if (messageBody.includes('pricing')) {
+      responseMessage = "See our pricing page https://papercups.io/pricing"
```

Comment:
might be nice to include example markdown, e.g. `[papercups.io/pricing](https://papercups.io/pricing)`